src/mobile/screens/ConfirmScreen/index.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { useIntl } from 'react-intl';
+import { useSelector } from 'react-redux';
+import { selectUserInfo } from '../../../modules/user/profile';
+import { ConfirmScreen } from '../../../screens/ConfirmScreen';
+import { Modal } from '../../components/Modal';
+
+const ConfirmMobileScreen = () => {
+    const intl = useIntl();
+    const user = useSelector(selectUserInfo) || { level: 1 };
```

Comment:
1. Default data for user already defined in redux reducer.
2. Default user level has not to be '1' in any case.